doc/readthedocs/sphinx/conf.py

```diff
@@ -1,32 +1,73 @@
 # -*- coding: utf-8 -*-
-#
-# Configuration file for the Sphinx documentation builder.
-#
-# This file does only contain a selection of the most common options. For a
-# full list see the documentation:
-# http://www.sphinx-doc.org/en/master/config
 
 from json import load
 import os
 from shutil import copytree, rmtree
 from subprocess import check_call, CalledProcessError, DEVNULL
 
 # -- Path setup --------------------------------------------------------------
-
-# If extensions (or modules to document with autodoc) are in another directory,
-# add these directories to sys.path here. If the directory is relative to the
-# documentation root, use os.path.abspath to make it absolute, like shown here.
 from pathlib import Path
 import sys
 this_file_path = Path(__file__)
 api_source_dir = this_file_path.parent.parent.parent.parent / 'src' / 'EnergyPlus' / 'api'
-print(f"**Adding api_source_dir: {api_source_dir}")
+print(f"* Adding api_source_dir: {api_source_dir}")
 sys.path.insert(0, str(api_source_dir))
+sys.path.insert(0, os.path.abspath('.'))
 
 # add a mock version of pyenergyplus
 autodoc_mock_imports = ["pyenergyplus"]
 
-# # build the C docs # #
+# -- Project information -----------------------------------------------------
+project = u'EnergyPlus'
+copyright = u'2023, National Renewable Energy Laboratory for the United States Department of Energy'
+author = u'National Renewable Energy Laboratory for the United States Department of Energy'
+version = u'23.1'  # The short X.Y version
+release = u'23.1'  # The full version, including alpha/beta/rc tags
+
+# -- General configuration ---------------------------------------------------
+extensions = [
+    'sphinx.ext.autodoc',
+    'sphinx.ext.coverage',
+    'sphinx.ext.mathjax',
+    'sphinx.ext.viewcode',
+]
+source_suffix = '.rst'
+master_doc = 'index'
+language = 'en'
+exclude_patterns = [u'_build', 'Thumbs.db', '.DS_Store']
+pygments_style = None
+
+# -- Options for HTML output -------------------------------------------------
+html_theme = 'sphinx_rtd_theme'
+html_logo = '../../../release/favicon_white_plus.png'
+html_show_sphinx = False
+html_show_copyright = True
+html_extra_path = ['_build_c']
+html_static_path = ['_static']
+html_css_files = [
+    'main.css',
+]
+html_js_files = [
+    'https://code.jquery.com/jquery-1.11.2.min.js',
+    'main.js',
+]
+extra_nav_links = {
+    'Blah': 'https://energyplus.net'
+}
+templates_path = ['.templates']
+html_theme_options = {
+    "titles_only": True,
+}
+# html_sidebars = {
+#     '**': [
+#         'about.html',
+#         'navigation.html',
+#         'relations.html',
+#         'searchbox.html',
+#     ]
+# }
+
```

Comment:
A lot of these changes are just rearranging, but I am also specifying some extra files to bring in during build.